src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OwnersAndOfficersDataBlockValidator.java

```diff
@@ -87,7 +87,8 @@ private boolean isCorrectCombinationOfOwnersAndOfficersForStatement(OverseasEnti
                  }
                  break;
             case SOME_IDENTIFIED_ALL_DETAILS:
-                if (!hasBeneficialOwners(overseasEntitySubmissionDto)) {
+                // TODO roe-1640 AC are that both at least one BO and MO must be present if either one is missing an error is logged and returned
+                if (!(hasBeneficialOwners(overseasEntitySubmissionDto) || hasManagingOfficers(overseasEntitySubmissionDto))) {
                     logValidationErrorMessage(errors, loggingContext, String.format("%s for statement that some can be identified", MISSING_BENEFICIAL_OWNER));
```

Comment:
Log message needs updating to reflect the new logic. Maybe something like:

`String.format("%s for statement that some can be identified", MISSING_BENEFICIAL_OWNER + " or " + MISSING_MANAGING_OFFICER));`

Reply:
Updated message as suggested.